src/examples/example-aleph-zero-unlinked.ts

```diff
@@ -0,0 +1,48 @@
+
```

Comment:
Can you add instructions on top here and in the other file like with the Terra example? And also the exact npx command to run